xpass/xpass.cc

```diff
@@ -211,6 +238,15 @@ void XPassAgent::recv_data(Packet *pkt) {
   update_rtt(pkt);
 }
 
+void XPassAgent::recv_nack(Packet *pkt) {
+  hdr_tcp *tcph = hdr_tcp::access(pkt);
+
+  struct packet_chunk * chunk = new struct packet_chunk;
```

Comment:
Please follow the convention. Either packet_chunk* chunk or packet_chunk *chunk.